scripts/db/migration-1.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE test_run ADD cmake_flags TEXT DEFAULT "NOT FOUND", ADD maxscale_source VARCHAR(256) DEFAULT "NOT FOUND";
+
+UPDATE test_run SET cmake_flags="NOT FOUND" WHERE cmake_flags IS NULL;
+UPDATE test_run SET maxscale_source="NOT FOUND" WHERE maxscale_source IS NULL;
+
+CREATE TABLE IF NOT EXISTS db_version (
```

Comment:
It is too expensive to create separate table for a versioning. Rename table to `db_metadata`.

Also there is no such table in the resulting database schema.